.github/config/conventional-bump-setup.js

```diff
@@ -0,0 +1,73 @@
+const COMMIT_TYPES = {
+  breaking: 'major',
+  feat: 'minor',
+  fix: 'patch',
+  perf: 'patch',
+  build: 'patch',
+  chore: 'patch',
+  refactor: 'patch',
+  docs: null,
+  style: null,
+  test: null,
+  ci: null,
+};
+
+const RELEASE_TYPES = {
+  0: 'MAJOR',
+  1: 'MINOR',
+  2: 'PATCH',
+};
+
+const config = {
+  whatBump: commits => {
```

Comment:
can you tell me why we need this? doesn't `conventional-recommended-bump` come with config out of the box?

Reply:
yes, it does have a config out of the box. but that config does not trigger a version bump for these commits:

```
  build
  chore
  refactor
  docs
  style
  test
  ci
```

Me and @shark0der had a discussion which commits we want to trigger a patch and we concluded we want `build`, `chore` and `refactor` to trigger a patch bump hence the custom config script.

This is probably something we as a team should discuss and should be in agreement @mixplore @ssbarbee @MilGard91 @duxxud @tomfutago @shark0der